_appmap/env.py

```diff
@@ -183,7 +208,7 @@ def _configure_logging(self):
                 "default": {
```

Comment:
If `determine_log_file` returns `None`, we don't need to add a handler here. Writing warnings to stderr is enough.